harbor-client/src/fedimint_client.rs

```diff
@@ -567,10 +567,8 @@ pub(crate) async fn spawn_lnv2_payment_subscription(
                     }
                     break;
                 }
-                SendOperationState::Success => {
+                SendOperationState::Success(preimage) => {
                     info!("Payment success");
-                    // TODO: Get preimage from state
```

Comment:
@m1sterc001guy Is this what you had in mind?

Reply:
Yup!